include/aikido/control/ros/RosTrajectoryExecutor.hpp

```diff
@@ -56,7 +55,8 @@ class RosTrajectoryExecutor : public aikido::control::TrajectoryExecutor
   ///
   /// To be executed on a separate thread.
   /// Regularly checks for the completion of a sent trajectory.
-  void step() override;
+  void step(
+      const std::chrono::system_clock::time_point& /*timepoint*/) override;
```

Comment:
Nit: We don't need to comment out for argument name in a header file; actually, we might want to have here so that Doxygen uses it. The unused variable warning happens only in the source file.